notebooks/intro.ipynb

```diff
@@ -343,7 +343,7 @@
     "- exposure_time\n",
     "- sky_brightness \n",
     "- zenith_psf_fwhm\n",
-    "- zero_points\n",
+    "- zeropoints\n",
```

Comment:
list to be updated according to the latest changes in `survey.py`

Reply:
yes, I will update notebooks later